src/js/lib/content/src/index.js

```diff
@@ -450,18 +429,28 @@ function ok(value) {
 
 /// Main entry point. Loads WASM, initializes it, chooses the scene to run.
 API.main = async function (inputConfig) {
-    remoteLog("main")
     let defaultConfig = {
-        use_loader    : true,
-        wasm_url      : '/assets/ide.wasm',
-        wasm_glue_url : '/assets/wasm_imports.js'
+        use_loader     : true,
+        wasm_url       : '/assets/ide.wasm',
+        wasm_glue_url  : '/assets/wasm_imports.js',
+        crashReportHost: cfg.defaultLogServerHost,
+        noDataGathering: false,
     }
     let urlParams = new URLSearchParams(window.location.search);
     let urlConfig = Object.fromEntries(urlParams.entries())
     let config    = Object.assign(defaultConfig,inputConfig,urlConfig)
     API[globalConfig.windowAppScopeConfigName] = config
 
-    //initCrashHandling()
+    API.remoteLog = config.noDataGathering
+                    ? function () {}
+                    : remoteLog
+    window.setInterval(() =>{API.remoteLog("alive");}, 1000 * 60)
```

Comment:
please expose the `60 seconds` as a constant variable on top of the file (in constants section)